pkg/kubecost/allocation.go

```diff
@@ -1464,6 +1511,7 @@ func (as *AllocationSet) AggregateBy(aggregateBy []string, options *AllocationAg
 
 	// (8) Distribute shared allocations according to the share coefficients.
 	if shareSet.Length() > 0 {
+		scBreakdown := SharedCostBreakdowns{}
```

Comment:
Shouldn't this be defined for each row? That would mean it should be instantiated within the `range aggSet.Allocations`, not before.